src/app/components/auth/auth.guard.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from "@angular/core";
```

Comment:
I created `auth.guard.ts` in `src/app/components/guards/auth.guard.ts`. You can delete this file.

Reply:
done. I deleted it.